torch/onnx/symbolic_opset9.py

```diff
@@ -2201,8 +2202,9 @@ def batch_norm(
         return res
 
 
-@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
-def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+def _layer_norm_shared(
+    g, input, normalized_shape, weight, bias, eps, cudnn_enable, return_mean_rstd
+):
```

Comment:
Annotate type for all params and return value

Reply:
I can't annotate it. As explained in dropout PR, it will trigger CI errors, so no exporter does it.

Reply:
Partially annotate after discussion. Thanks.